.changeset/olive-tomatoes-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@manypkg/cli": minor
+---
+
+Node requiremet raised to `node >= 18.18.0`. Upgraded `package-json`, `semver` and `sembear` dependencies.
```

Comment:
the updated dependencies should be one changeset file and the bumped node requirement should be another. Even though they are introduced by a single PR they will like 2 separate changes for the end user

Reply:
Thanks, I'm not used to writing changesets so this is very helpful!